scripts/release-lib.sh

```diff
@@ -120,7 +120,10 @@ function commit_changelog() {
 function update_copyright() {
   version=$1
 
-  if ! "$scripts_folder/update-copyright"
+  DIR=$(dirname "$0")
+  PDIR=$(dirname "$DIR")
+
+  if ! (docker build -t kong/update-copyright ${DIR} && docker run -v ${PDIR}:/workspace --rm kong/update-copyright)
```

Comment:
We may consider publishing the docker image to reduce the build time in the future. But for now I've tried it several times docker always uses the local cached image thus it's not a big deal. @kikito any preferences?